evms/marshalling.go

```diff
@@ -25,6 +25,20 @@ func CustomMarshal(log *opLog) []byte {
 	b = append(b, []byte(`,"pc":`)...)
 	b = strconv.AppendUint(b, uint64(log.Pc), 10)
 
+	// Depends on Geth EOF support
+	// Even then, not essential to diffing
```

Comment:
It doesn't really depend on geth support. You can do like I do with the globals, e.g `ClearGascost`. 
```
	// Besu sometimes reports GasCost of 0x7fffffffffffffff, along with ,"error":"Out of gas"
	ClearGascost = true
```
So you add `IgnoreEOF = true`, in the constants, and 
```golang
if !IgnoreEOF {
	if log.Section != 0 {
		b = append(b, []byte(`,"section":`)...)
		b = strconv.AppendUint(b, uint64(log.Section), 10)
	}
	if log.FunctionDepth ... 
```

Reply:
Will do.  Some of the commented out code depends on types from go-ethereum code.  Those will remain commented out.